### PR TITLE
Prepare 0.16.0 release

### PR DESCRIPTION
**Pull request by tigrannajaryan**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,14 @@ Full list of differences found in [this compare](https://github.com/open-telemet
...
 
+* Remove if no changes for this section before release.
+
+## 0.16.0
+
+Full list of differences found in [this compare](https://github.com/open-telemetry/opentelemetry-proto/compare/v0.15.0...v0.16.0).
+
+### Removed
+
 * Remove deprecated LogRecord.Name field.
```
Comment (arminru) [inline CHANGELOG.md:29]:
```suggestion
* Remove deprecated LogRecord.Name field (#373).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -18,6 +18,14 @@ Full list of differences found in [this compare](https://github.com/open-telemet
 
 ### Removed
 
+* Remove if no changes for this section before release.
+
+## 0.16.0
```
Comment (arminru) [inline CHANGELOG.md:23]:
```suggestion
## 0.16.0 - 2022-03-31
```

**Review by arminru — APPROVED** [review]

**Review by jmacd — APPROVED** [review]

**Review by codeboten — APPROVED** [review]

**Review by carlosalberto — APPROVED** [review]

**Comment by jmacd** [comment]
I propose to merge #279 after this PR. Are we waiting for #376?

**Comment by tigrannajaryan** [comment]
@jmacd I will release 0.16.0 so that you can merge https://github.com/open-telemetry/opentelemetry-proto/pull/279.

We are not waiting for https://github.com/open-telemetry/opentelemetry-proto/pull/376 I want to give it a bit more time for people to see and comment.

**Comment by tigrannajaryan** [comment]
0.16.0 released.

**Pull request merged**